web/assets/js/talk/listeners.js

```diff
@@ -0,0 +1,44 @@
+//Favorite Listener
+$('.js-talk-favorite').on('click', function(e) {
+  var talk = new Talk($(this).data('id'), $(this));
+  e.preventDefault();
+  talk.favorite();
+});
+
+//Select Listener
+$('.js-talk-select').on('click', function(e) {
+  var talk = new Talk($(this).data('id'), $(this));
+  e.preventDefault();
+  talk.select();
+});
+
+//Rating Listeners
+//Yes No Rating listener
+$('.js-talk-rating-yes-no').on('click', function(e) {
+  var id = $(this).data('id');
+  var talk = new Talk($(this).data('id'), $(this), $(this).data());
+  var rating = $(this).data('rating');
+  e.preventDefault();
+  if (talk.rate(rating)) {
+    if (rating === -1) {
+      $('#talk-downvote-' + id + ' i').addClass('text-red-dark');
```

Comment:
@BackEndTea @mdwheele 

What do you think about using more semantic CSS classes (in a separate PR)?